pkg/output/output.go

```diff
@@ -48,29 +48,46 @@ func (f *fsWriter) Writer(parentPath, name string, permissions os.FileMode) (Clo
 	}, writer, nil
 }
 
-// NewFSWriter returns an Outputter which writes the files and directories to a specified location.
-func NewFSWriter(path string) (Outputter, error) {
+func EnsureOutputPath(path string, deleteIfExists bool) error {
 	info, err := os.Stat(path)
 	if err != nil {
 		if !os.IsNotExist(err) {
-			return nil, err
+			return err
 		} else {
 			err := os.Mkdir(path, 0700)
```

Comment:
You can return at this point and get rid of the else branch.

Reply:
done!